src/main/jhipster/src/main/webapp/app/shared/util/date-utils.ts

```diff
@@ -0,0 +1,7 @@
+import * as moment from 'moment';
+
+export const DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
+
+export function parseDate(date: number): string {
+    return moment(date).format(DATE_FORMAT);
+}
```

Comment:
Nice I'll try to cherry pick this, I need it to validate variable type dates in germp import

Reply:
https://github.com/IntegratedBreedingPlatform/Workbench/commit/8f6f6ea6a940d89c760fbdf10fde480e789e77e2